ReactAndroid/src/main/java/com/facebook/react/uimanager/events/TouchEvent.java

```diff
@@ -67,30 +69,33 @@ private void init(
       int viewTag,
       TouchEventType touchEventType,
       MotionEvent motionEventToCopy,
+      long gestureStartTime,
       float viewX,
       float viewY,
       TouchEventCoalescingKeyHelper touchEventCoalescingKeyHelper) {
     super.init(viewTag);
 
+    Assertions.assertCondition(gestureStartTime != Long.MIN_VALUE,
```

Comment:
Sorry this is a bit confusing, but Assertions.assertCondition will throw an AssertionError which will hardcrash the app -- I'd prefer we use SoftAssertions from the .common package which throws an Exception instead.